docs/home/contribute/includes.md

```diff
@@ -1,12 +1,66 @@
 ---
 approvers:
 - chenopis
-title: Tabs Example
+title: Standard Include Statements
 ---
 
+{% capture overview %}
+This page explains the special include statements that can be used in
+Kubernetes documentation markdown.
+{% endcapture %}
+
+{% capture body %}
+## Feature state
+
+In a markdown page (.md file) on this site, you can add a tag to display
+version and state of the documented feature.
+
+### Feature state demo
+
+{% assign for_k8s_version = "1.6" %}
+{% include feature-state-stable.md %}
+
+### Feature state code
+
+Below is the template code for each available feature state.
+
+The displayed Kubernetes version defaults to that of the page. This can be
+changed by setting the <code>for_k8s_version</code> variable.
+
+````liquid
+{{ "{% assign for_k8s_version = " }} "1.6" %}
+{{ "{% include feature-state-stable.md " }}%}
+````
+
+#### Alpha feature
+
+````liquid
+{{ "{% include feature-state-alpha.md " }}%}
+````
+
+#### Beta feature
+
+````liquid
+{{ "{% include feature-state-beta.md " }}%}
+````
+
+#### Stable feature
+
+````liquid
+{{ "{% include feature-state-stable.md " }}%}
+````
+
+#### Deprecated feature
+
+````liquid
+{{ "{% include feature-state-deprecated.md " }}%}
+````
+
+## Tabs
+
 In a markdown page (.md file) on this site, you can add a tab set to display multiple flavors of a given solution. 
 
-## Demo
+### Tabs demo
```

Comment:
I'd suggest putting a single sentence at the beginning of this demo (and the feature state demo) providing context. e.g.

> Below is a demo of the tabs feature. Here it is used to display each of the installation commands for the various Kubernetes network solutions.

Reply:
:white_check_mark: